AFNetworking/AFNetworking
# Fixed overwrite a file

Hi,

You have problems in AFURLSessionManager
You can't overwrite a file if it exist

This can move file if it doesn't exist:
```
[[NSFileManager defaultManager] moveItemAtURL: toURL: error:];
```

Test:
1. Start download a file
2. Update file on server
3. Start download a file
4. Check file
